src/sentry/integrations/issues.py

```diff
@@ -298,18 +298,9 @@ class IssueSyncMixin(IssueBasicMixin):
     outbound_assignee_key = None
     inbound_assignee_key = None
 
-    def should_sync(self, attribute):
-        try:
-            key = getattr(self, "%s_key" % attribute)
-        except AttributeError:
-            return False
-
-        if key is None:
-            return False
-
-        config = self.org_integration.config
-
-        return config.get(key, False)
+    def should_sync(self, attribute: str) -> bool:
+        key = getattr(self, f"{attribute}_key", None)
+        return self.org_integration.config.get(key, False)
```

Comment:
I assume there's no chance there's a `None` key in this config that would cause this to return something different than before?

Reply:
I assume so too but I'll add back a null check.